md_question_one.ipynb

```diff
@@ -0,0 +1,1005 @@
+{
```

Comment:
<p>praise: this is good for question 3 as you mentioned :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepakkrish212/MATH327-Final/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>